docs/troubleshoot/troubleshoot-zowe-release.md

```diff
@@ -12,18 +12,25 @@ Each time a new release is created, the release ID is incremented.  Each number
 
-When Zowe is version one, it is associated with the Zowe v1 [conformance program](../extend/zowe-conformance-program.md). Offerings that extend Zowe and achieve the Zowe v1 conformance badge will remain compatible with Zowe throughout its version 1 lifetime. A major release increment because of incompatibility is sometimes referred to as a "breaking" change.
+A major release is required if changes are made to the public API and the code is no longer compatible with an earlier version.  **[is this correct? what is no api changes are made? is it ok to just remove this sentence?]**
 
-The first SMP/E build for Zowe v2 has a Functional Module ID (FMID) of AZWE002, which was created with content from the 2.0.0 release. Each major release will be its own SMP/E FMID where the last digit is updated, for example AZWE00V where V represents the major version.
+Major releases are required for a *"breaking" change*, or a modification that requires updates to avoid disruptions in your applications. Major releases also can be used to indicate to the community a significant content update over and above what would be included in a minor release.
 
-Subsequent minor and patch releases to V2 are delivered as SMP/E PTF SYSMODs.  Because of the size of the content, two co-requisite PTFs are created for each Zowe release.
+#### Conformance programs
 
-While Major releases are required for a "breaking" change, they also can be used to indicate to the community a significant content update over and above what would be included in a minor release.
+Zowe V1 is associated with the Zowe V1 [conformance program](../extend/zowe-conformance-program.md). Offerings that extend Zowe and achieve the Zowe V1 conformance badge remain compatible with Zowe throughout its Version 1 lifetime. A major release increment because of incompatibility is sometimes referred to as a "breaking" change.
+
+#### SMP/E builds
+
+ Each major release has its own SMP/E Functional Module ID (FMID) in the format `AZWE00V`, where `V` represents the major version. The first SMP/E build for Zowe V3 has a Functional Module ID (FMID) of `AZWE003`, which was created with content from the 3.0.0 release.
+
+Subsequent minor and patch releases to V3 are delivered as SMP/E PTF SYSMODs.  Because of the size of the content, two co-requisite PTFs are created for each Zowe release.
 
 ### Minor release
-A minor release indicates that new functionality is added but the code is compatible with an earlier version.  The Zowe community works on two-week sprints and creates a minor release at the end of these, typically once per month although the frequency might vary.
+
+A minor release indicates that new functionality is added but the code is compatible with an earlier version.  The Zowe community works on two-week sprints and creates a minor release at the end of these, typically once per month although the frequency might vary. **[is this correct?]**
```

Comment:
@adam-wolfe - please respond

Reply:
I would remove "The Zowe community works on two-week sprints and creates a minor release at the end of these, typically once per month although the frequency might vary."

Not all squads work in sprints and we the release schedule is a bit more complicated than saying we release minor versions every x months.

For the release schedule, we could point them to https://github.com/zowe/community/blob/master/Project%20Management/Schedule/Zowe%20PI%20%26%20Sprint%20Cadence.md